### ISSUE TITLE
Lotus daemon cannot run。

### ISSUE THREAD

**Issue by kangqian623** [filecoin-project/lotus#1878]
**Describe the bug**
the vlog file in badger dir is so big that cause an error in lotus daemon
**To Reproduce**
1. Run 'lotus daemon'
2. See error

**Expected behavior**
clear vlog file and make sure daemon can run 

**Screenshots**
```
2020/05/28 09:36:08 file size: 1412968288 greater than 4294967295
github.com/dgraph-io/badger/v2/y.AssertTruef
 /home/qq/go/pkg/mod/github.com/dgraph-io/badger/v2@v2.0.3/y/error.go:62
github.com/dgraph-io/badger/v2.(*logFile).open
 /home/qq/go/pkg/mod/github.com/dgraph-io/badger/v2@v2.0.3/value.go:910
github.com/dgraph-io/badger/v2.(*valueLog).open
 /home/qq/go/pkg/mod/github.com/dgraph-io/badger/v2@v2.0.3/value.go:1120
github.com/dgraph-io/badger/v2.Open
 /home/qq/go/pkg/mod/github.com/dgraph-io/badger/v2@v2.0.3/db.go:380
github.com/ipfs/go-ds-badger2.NewDatastore
 /home/qq/go/pkg/mod/github.com/ipfs/go-ds-badger2@v0.1.0/datastore.go:137
github.com/filecoin-project/lotus/node/repo.(*fsLockedRepo).Datastore.func1
 /home/qq/桌面/lotus/node/repo/fsrepo.go:268
sync.(*Once).doSlow
 /usr/lib/go-1.14/src/sync/once.go:66
sync.(*Once).Do
 /usr/lib/go-1.14/src/sync/once.go:57
github.com/filecoin-project/lotus/node/repo.(*fsLockedRepo).Datastore
 /home/qq/桌面/lotus/node/repo/fsrepo.go:264
```

**Version (run `lotus --version`):**
lotus version 0.3.0'+git9a0d0511'

**Additional context**

detail see the code 

```
func (lf *logFile) open(path string, flags uint32) error {
	var err error
	if lf.fd, err = y.OpenExistingFile(path, flags); err != nil {
		return y.Wrapf(err, "Error while opening file in logfile %s", path)
	}

	fi, err := lf.fd.Stat()
	if err != nil {
		return errFile(err, lf.path, "Unable to run file.Stat")
	}
	sz := fi.Size()
	y.AssertTruef(
		sz <= math.MaxUint32,
		"file size: %d greater than %d",
		uint32(sz), uint32(math.MaxUint32),
	)
	lf.size = uint32(sz)
	return nil
}

```

**Comment by jimpick** [comment]
I hit the same issue in #1831 ...

**Comment by roiger** [comment]
i have the same issue.some one like has deal it.
https://github.com/filecoin-project/lotus/pull/1879
It works

**Comment by arajasek** [comment]
Fixed as part of the work going into #2458
